eng/mgmt/automation/generate.py

```diff
@@ -199,6 +207,117 @@ def sdk_automation_autorest(config: dict) -> List[dict]:
     return packages
 
 
+def sdk_automation_typespec(config: dict) -> List[dict]:
+
+    packages = []
+    if 'relatedTypeSpecProjectFolder' not in config:
+        return packages
+
+    tsp_projects = config['relatedTypeSpecProjectFolder']
+    if isinstance(tsp_projects, str):
+        tsp_projects = [tsp_projects]
+
+    for tsp_project in tsp_projects:
+        # mgmt tsp project folder follow the pattern, e.g. specification/deviceregistry/DeviceRegistry.Management
+        if re.match(r'specification[\\/](.*)[\\/](.*)[\\.]Management', tsp_project):
+            packages.append(sdk_automation_typespec_project(tsp_project, config))
+        else:
+            packages.append(sdk_automation_typespec_project_data(tsp_project, config))
```

Comment:
support 
```json
"relatedTypeSpecProjectFolder": [
    "specification/deviceregistry/DeviceRegistry.Management",
    "specification/cognitiveservices/ContentSafety"
]
```

though in real life, automation pipeline passed in two separate `generateInput.json` file...

Reply:
I don't think this is what we need to support.

The configure is once invocation per project.

If these is multiple project, the output files would be mixed and probably not able to do a clean PR (client_version etc.)